### PR TITLE
Fixed long strip splitting method causing flickering black line

### PR DESCRIPTION
**Pull request by Saud-97**
I noticed an issue with current splitting method where in webtoons there can be a flickering black line between pages when reading long strip pages. I updated to upstream method which doesn't have that issue.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt [hunk 1]
```diff
@@ -586,9 +557,12 @@ object ImageUtil {
             return false
         }
 
-        Timber.d(
-            "Splitting image with height of $imageHeight into $partCount part with estimated ${optimalSplitHeight}px height per split",
-        )
+        val options = extractImageOptions(imageFile.openInputStream(), resetAfterExtraction = false).apply {
+            inJustDecodeBounds = false
+        }
+        val imageWidth = options.outWidth
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt:563]: no real need to define this, either below you can do

`val region = Rect(0, splitData.topOffset, options.outWidth, splitData.bottomOffset)`

or

`val region = Rect(0, splitData.topOffset, splitData.splitWidth, splitData.bottomOffset)`

File: app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt [hunk 2]
```diff
@@ -601,9 +575,6 @@ object ImageUtil {
                     splitBitmap.compress(Bitmap.CompressFormat.JPEG, 100, outputStream)
                     splitBitmap.recycle()
                 }
-                Timber.d(
-                    "Success: Split #${splitData.index + 1} with topOffset=${splitData.topOffset} height=${splitData.outputImageHeight} bottomOffset=${splitData.bottomOffset}",
-                )
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt:606]: why was this removed?

File: app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt [hunk 3]
```diff
@@ -586,9 +557,12 @@ object ImageUtil {
             return false
         }
 
-        Timber.d(
-            "Splitting image with height of $imageHeight into $partCount part with estimated ${optimalSplitHeight}px height per split",
-        )
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt:591]: why is this removed too?

**Comment by AntsyLich** [comment]
I would appreciate it if you gave me coauthor

File: app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt (hunk 3 above)
Comment (Saud-97) [reply app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt:591]: I was intending to move it down to the new method for splitting.

File: app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt (hunk 1 above)
Comment (Saud-97) [reply app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt:563]: Ok

**Comment by Saud-97** [comment]
> I would appreciate it if you gave me coauthor

Done, I thought I already did.
